packages/components/molecules/f-tabs/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@justeat/f-tabs",
+  "description": "Fozzie Tabs – Switchable slots for content&#34;",
```

Comment:
Is the `&#34` meant to be there?

Reply:
it was generated this way, will change but can we fix this in generator?

Reply:
Yeh, I'll take a look at that 👍